README.md

```diff
@@ -25,14 +25,14 @@ We are moving development of Fractal v2.0 and all first-party add-ons into a 'mo
 #### Running locally
 
 1. Clone this repository
-2. Install dependencies - `npm install`
-3. Bootstrap the packages together using Lerna - `npm run bootstrap`
+2. Install dependencies - `yarn install`
```

Comment:
Ok, shall I swap out to npm for scripts then, and revert the Readme?
Re. the pretest script, only issue with that is that it would runs every time I ran the test locally as well. Can we create a separate test:ci script, or something to that effect?